"Failed to read pull request template"

### Version

0.13.0

### Operating System

Windows

### Distribution Method

msi (Windows)

### Describe the issue

I selected a PR template, but I get

![grafik](https://github.com/user-attachments/assets/aa4f20c5-9ea9-4e3a-b11e-226e8f9b1981)

I checked in Explorer - the file exists:

![grafik](https://github.com/user-attachments/assets/bacac520-5599-4a39-a41b-692640a401d6)

---

Think, this refs https://github.com/gitbutlerapp/gitbutler/pull/4814

### How to reproduce

1. Clone JabRef repo
2. Enable pull request templates\
    ![grafik](https://github.com/user-attachments/assets/fbe1f2e4-58ca-4687-9086-79b370a129c5)
3. Close settings
4. See that error pop up that PR template is not found

### Expected behavior

PR template is used.


### Relevant log output

_No response_

I get the same error. Windows. Version 0.13.0
```
ipc->get_pr_template_contents: {"relativePath":"D:\\source\\MewsSystems\\guest-profiles\\.github\\PULL_REQUEST_TEMPLATE.md","projectId":"a853f7a3-7ad0-4aa3-8f62-79a1f0ff7015"} a: Invalid workspace file
    at a.fromError (https://tauri.localhost/_app/immutable/chunks/Icon.Bfycfzga.js:1:1104)
    at S1 (https://tauri.localhost/_app/immutable/chunks/Icon.Bfycfzga.js:1:1310)
    at async Za.pullRequestTemplateContent (https://tauri.localhost/_app/immutable/nodes/3.IgIM1k96.js:201:8251) Object
(anonymous) @ user.CSv90AG6.js:4
user.CSv90AG6.js:4  Error reading pull request template at path: D:\source\MewsSystems\guest-profiles\.github\PULL_REQUEST_TEMPLATE.md a: Invalid workspace file
    at a.fromError (https://tauri.localhost/_app/immutable/chunks/Icon.Bfycfzga.js:1:1104)
    at S1 (https://tauri.localhost/_app/immutable/chunks/Icon.Bfycfzga.js:1:1310)
    at async Za.pullRequestTemplateContent (https://tauri.localhost/_app/immutable/nodes/3.IgIM1k96.js:201:8251)
```

--- a/crates/gitbutler-project/src/controller.rs
+++ b/crates/gitbutler-project/src/controller.rs
@@ -72,7 +72,7 @@ impl Controller {
         let project = Project {
             id: ProjectId::generate(),
             title,
-            path: path.to_path_buf(),
+            path: gix::path::realpath(path)?,
             api: None,
             ..Default::default()
         };

--- a/crates/gitbutler-repo/src/commands.rs
+++ b/crates/gitbutler-repo/src/commands.rs
@@ -49,25 +49,24 @@ impl RepoCommands for Project {
 
     fn read_file_from_workspace(&self, relative_path: &Path) -> Result<String> {
         let ctx = CommandContext::open(self)?;
-        if self
-            .path
-            .join(relative_path)
-            .canonicalize()?
-            .as_path()
-            .starts_with(self.path.clone())
-        {
-            let tree = ctx.repository().head()?.peel_to_tree()?;
-            let entry = tree.get_path(relative_path)?;
-            let blob = ctx.repository().find_blob(entry.id())?;
+        let path_in_worktree = gix::path::realpath(self.path.join(relative_path))?;
+        if !path_in_worktree.starts_with(self.path.clone()) {
+            anyhow::bail!(
+                "Path to read from at '{}' isn't in the worktree directory '{}'",
+                relative_path.display(),
+                self.path.display()
+            );
+        }
 
-            if !blob.is_binary() {
-                let content = std::str::from_utf8(blob.content())?;
-                Ok(content.to_string())
-            } else {
-                anyhow::bail!("File is binary");
-            }
+        let tree = ctx.repository().head()?.peel_to_tree()?;
+        let entry = tree.get_path(relative_path)?;
+        let blob = ctx.repository().find_blob(entry.id())?;
+
+        if !blob.is_binary() {
+            let content = std::str::from_utf8(blob.content())?;
+            Ok(content.to_string())
         } else {
-            anyhow::bail!("Invalid workspace file");
+            anyhow::bail!("File is binary");
         }
     }
 }

--- a/crates/gitbutler-testsupport/src/testing_repository.rs
+++ b/crates/gitbutler-testsupport/src/testing_repository.rs
@@ -1,5 +1,6 @@
 use std::fs;
 
+use crate::init_opts;
 use gitbutler_oxidize::git2_to_gix_object_id;
 use gix_testtools::bstr::ByteSlice as _;
 use tempfile::{tempdir, TempDir};
@@ -13,7 +14,7 @@ pub struct TestingRepository {
 impl TestingRepository {
     pub fn open() -> Self {
         let tempdir = tempdir().unwrap();
-        let repository = git2::Repository::init(tempdir.path()).unwrap();
+        let repository = git2::Repository::init_opts(tempdir.path(), &init_opts()).unwrap();
 
         let config = repository.config().unwrap();
         match config.open_level(git2::ConfigLevel::Local) {

